projects/Examples/example4/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required (VERSION 2.6)
 
 
```

Comment:
replace by:
if (NOT DEFINED LEMONADE_INCLUDE_DIR)
    message("LEMONADE_INCLUDE_DIR is not provided. Build might fail. Use -DLEMONADE_INCLUDE_DIR=/path/to/installed/include/header/LeMonADE/ or install to default location")
endif()

if (NOT DEFINED LEMONADE_LIBRARY_DIR)
    message("LEMONADE_LIBRARY_DIR is not provided. Build might fail. Use -DLEMONADE_LIBRARY_DIR=/path/to/installed/lib/LeMonADE/ or install to default location")
endif()

include_directories (${LEMONADE_INCLUDE_DIR})
link_directories (${LEMONADE_LIBRARY_DIR})